_config.yml

```diff
@@ -134,17 +134,42 @@ keep_files:
 
 # Plug-ins
 plugins:
+  - jekyll-archives
+  - jekyll-feed
+  - jekyll-sitemap
   - jekyll-email-protect
   - jekyll-github-metadata
-  - jekyll-feed
   - jekyll-paginate-v2
   - jekyll/scholar
   - jekyll-twitter-plugin
   - jemoji
 
+# Sitemap settings
+defaults:
+  -
+    scope:
+      path:            "assets/**/*.*"
+    values:
+      sitemap:         false
+
 # Extras
 github: [metadata]
 
+# -----------------------------------------------------------------------------
+# Jekyll Archives
+# -----------------------------------------------------------------------------
+
+jekyll-archives:
+  enabled: [year, tags, categories]
+  layouts:
+    year: year-archive
+    tag: tag-archive
+    category: category-archive
+  permalinks:
+    year: '/:year/'
+    tag: '/tag/:name/'
+    category: '/category/:name/'
```

Comment:
let's change permalinks to `/blog/...` so that everything gets nested under `/blog/`.